engine/class_modules/sc_hunter.cpp

```diff
@@ -5531,9 +5542,19 @@ struct kill_command_t: public hunter_spell_t
       pet -> active.kill_command -> execute_on_target( target );
 
     if ( p() -> tier_set.t31_bm_4pc.ok() )
```

Comment:
you should add brackets on this if statement so it's less ambiguous what happens there

Reply:
good spot, I missed that

Reply:
good spot, I missed that.